features/step_definitions/tag_steps.rb

```diff
@@ -124,6 +124,17 @@
   end
 end
 
+Given /^a period-containing tag "([^\"]*)" with(?: (\d+))? comments$/ do |tagname, n_comments|
+  tag = Fandom.find_or_create_by_name(tagname)
```

Comment:
Aha, yes. And we need to do the create if it's not already there, but it doesn't need to be assigned to a variable.